src/core/gs.cpp

```diff
@@ -783,22 +783,87 @@ void GraphicsSynthesizer::render_line()
     }
 }
 
+int32_t GraphicsSynthesizer::orient2D(const Point &v1, const Point &v2, const Point &v3)
+{
+    return (v2.x - v1.x) * (v3.y - v1.y) - (v3.x - v1.x) * (v2.y - v1.y);
+}
+
 void GraphicsSynthesizer::render_triangle()
 {
     printf("\n[GS] Rendering triangle!");
-    int32_t point[3];
     uint32_t color = 0x00000000;
     color |= vtx_queue[0].rgbaq.r;
     color |= vtx_queue[0].rgbaq.g << 8;
     color |= vtx_queue[0].rgbaq.b << 16;
     color |= vtx_queue[0].rgbaq.a << 24;
-    color = 0xFFFFFFFF;
-    for (int i = 2; i >= 0; i--)
+
```

Comment:
Perhaps it might be useful to indicate which algorithm this code uses? This is so that outsiders can more easily understand what's going on.